react-paw-mailmerge/src/components/table/TableRow.tsx

```diff
@@ -23,7 +26,7 @@ const TableRow: React.FC<TableRowProps> = ({rowData, header, maxColumnCount, tem
                 (colName) => (<td key={colName}>{rowData[colName]}</td>)
                 )}
             <td>
-                <Button type='button' onClick={handleViewResult}>View Result</Button>
+            <Button type="button" {...errorAttributes} onClick={handleViewResult}>View Result</Button>
```

Comment:
That's a cool way to add attributes to a JSX element.

Reply:
Thanks. Took me way too long to figure it out.